### PR TITLE
[SP-11322] CLI upgrade everything

### PR DESCRIPTION
**Pull request by eeerin**
We dont need printHelp or unknownCommandWrapper any more those are built into the commander library now. Jumping from version 2 to 10 oof it was a little rough.

### CODE REVIEW

**Review by spacetc62 — CHANGES_REQUESTED** [review]
I *think* you want to revert all your `.storeOptionsAsProperties()`, and that will make the action arguments make more sense. See their tests here:
https://github.com/tj/commander.js/blob/8b03ab75b5431fd2d58a24b842ba088d621f12dc/tests/commander.configureCommand.test.js#L58

It looks like when storeOptionsAsProperties is false, action arguments are `name, options, command` but when its true they are `name, command, command`

File: commands/utils/export.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const { getExporter } = require('../../lib');
+const { getExporter } = require('../../lib/');
```
Comment (spacetc62) [inline commands/utils/export.js:1]: any particular reason?

File: lib/experience-bootstrap.js [hunk 2]
```diff
@@ -9,10 +9,10 @@ const getDownloader = require('./get-downloader');
 const downloader = getDownloader(experience);
 const inquirer = require('inquirer');
 
-module.exports = async (cmd, loadedConfig, applicationInfo) => {
+module.exports = async (cmd, cmd2, loadedConfig, applicationInfo) => {
```
Comment (spacetc62) [inline lib/experience-bootstrap.js:12]: I think you should still rename these

File: lib/experience-layout.js [hunk 3]
```diff
@@ -13,12 +13,13 @@ const c = require('chalk');
 const VIEW_HEADERS = [ 'Page Name', 'Layout Name' ];
 
 module.exports = async (page, command) => {
+  const opts = command.opts ? command.opts() : {};
   const { apiToken, applicationId, api, appUrl } = await loadConfig();
   if (!applicationId || !apiToken) { return; } // config did not lock or did not load correctly.
   const pageQuery = { applicationId, viewType: 'page' };
-  if (page || command.pattern) {
+  if (page || opts.list) {
```
Comment (spacetc62) [inline lib/experience-layout.js:20]: why the change from pattern -> list?

**Review by spacetc62 — CHANGES_REQUESTED** [review]
I think there are other libraries you could upgrade, but can do that in other PRs

We should probably also move away from `request`?

File: lib/experience-layout.js (hunk 3 above)
Comment (eeerin) [reply lib/experience-layout.js:20]: so the option is `.option('-l, --list <pattern>', 'pages that match this pattern will be listed with their layout')` I dont think this ever worked? or it got changed in the upgrade. but either way it should be list not pattern.

**Comment by eeerin** [comment]
inquirer and chalk cannot be upgraded because they are strictly esm
js-yaml has breaking changes with loading yaml and we use this version on the platform
error cannot be upgraded
omnibelt is not upgradable.

**Comment by eeerin** [comment]
We should probably also move away from request?

- filed a ticket for this.

**Review by spacetc62 — APPROVED** [review]
